### PR TITLE
Updated matomo-tests.yml.twig template to install ripgrep and testomat key

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Please include a description of this change and which issue it fixes. If no issue exists yet please include context and what problem it solves.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by caddoo — APPROVED** [review]

File: plugins/TestRunner/templates/matomo-tests.yml.twig [hunk 1]
```diff
@@ -220,6 +222,7 @@ jobs:
         uses: matomo-org/github-action-tests@main
         with:
 {% if plugin %}
+          testomatio: ${{ '{{secrets.TESTOMATIO}}' }}
```
Comment (AltamashShaikh) [inline plugins/TestRunner/templates/matomo-tests.yml.twig:225]: @achakko FYI new plugins in future, will have this line automatically from next release

**Comment by AltamashShaikh** [comment]
@caddoo Can you merge this ?

**Review by michalkleiner — APPROVED** [review]

File: plugins/TestRunner/templates/matomo-tests.yml.twig (hunk 1 above)
Comment (achakko) [reply plugins/TestRunner/templates/matomo-tests.yml.twig:225]: Awesome, thanks much @AltamashShaikh !

File: plugins/TestRunner/templates/matomo-tests.yml.twig [hunk 2]
```diff
@@ -98,6 +98,8 @@ jobs:
 {% if hasSubmodules %}
           submodules: true
 {% endif %}
+      - name: Install package ripgrep
+        run: sudo apt-get install ripgrep
```
Comment (sgiehl) [inline plugins/TestRunner/templates/matomo-tests.yml.twig:102]: Wouldn't it have been smarter to add that to the github action in `matomo-org/github-action-tests`, that way it' wouldn't have been required to add it to every single plugin

Comment (michalkleiner) [reply plugins/TestRunner/templates/matomo-tests.yml.twig:102]: Yeah, not sure why it didn't cross my mind then as I wasn't a huge fan of adding it here.

@AltamashShaikh keen to move it?

Comment (AltamashShaikh) [reply plugins/TestRunner/templates/matomo-tests.yml.twig:102]: @sgiehl I wasn't aware if that was possible, I would have added it there definitely and it wold have saved alot of time for me, @michalkleiner I can create a PR to move this in `github-action-tests`

**Pull request merged**